configure.ac

```diff
@@ -146,13 +146,15 @@ AC_CONFIG_FILES([
 	lib/libspl/asm-i386/Makefile
 	lib/libspl/asm-x86_64/Makefile
 	lib/libspl/include/Makefile
-	lib/libspl/include/ia32/Makefile
-	lib/libspl/include/ia32/sys/Makefile
 	lib/libspl/include/os/Makefile
 	lib/libspl/include/os/freebsd/Makefile
 	lib/libspl/include/os/freebsd/sys/Makefile
+	lib/libspl/include/os/freebsd/ia32/Makefile
```

Comment:
Please keep sorted.

Reply:
It seems my brain secretly prefers neat increasing lengths, over alphabetical order. Updated!